starboard/extension/extension_test.cc

```diff
@@ -590,5 +591,27 @@ TEST(ExtensionTest, MediaSettings) {
       << "Extension struct should be a singleton";
 }
 
+TEST(ExtensionTest, CobaltAccessibilityExtension) {
```

Comment:
The tests could and should be more extensive. The original test in `starboard/nplb/accessibility_test.cc` is about 150 lines of code. 

Please copy and convert that code as well. It does not need to be in the same `.cc` module, adding it int `starboard/extension/accessibilty_extension_test.cc` is fine. 

In the interest of getting Starboard 16 finalized, it's fine to submit this without the tests, but we need to follow up.